File: redsheet-portal/src/app/database-models/v-lite-meeting-planning.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vLiteMeetingPlanning {
    constructor(vLiteMeetingPlanning?: vLiteMeetingPlanning )  {
        let properties = [ "liteMeetingManagementId", "liteMeetingPlanningId", "liteMeetingPlanningValue",];
        for (let property of properties)
            if (vLiteMeetingPlanning && vLiteMeetingPlanning[property])
                this[property] = vLiteMeetingPlanning[property];
    }
 
	liteMeetingManagementId : number =   0 ;
 
    @range(1,2147483647)
	liteMeetingPlanningId : number =   undefined;
 
    @required()
    @maxLength(1000)
	liteMeetingPlanningValue : string =   undefined;


}

File: RedSheet.DbEntities/Models/vLiteMeetingPlanning.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vLiteMeetingPlannings", Schema = "dbo")]
	[Serializable]
	public partial class vLiteMeetingPlanning  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int LiteMeetingManagementId { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Range(1, int.MaxValue)]
	public int LiteMeetingPlanningId { get; set; } 
 
	[Required] 
	[MaxLength(1000)]
	public string LiteMeetingPlanningValue { get; set; } 

	public vLiteMeetingPlanning()
	{
    }

    }
}

